inc/class-cachify.php

```diff
@@ -270,6 +294,14 @@ public function __construct() {
 	 * @change  2.1.0
 	 */
 	public static function on_deactivation() {
+		/* Remove hdd cache cron when hdd is selected */
+		if ( self::METHOD_HDD == self::$options['use_apc'] ) {
```

Comment:
Same, === instead of ==

Reply:
Is included in "Make requested changes to checks"